src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.cs

```diff
@@ -183,6 +183,12 @@ public override void VisitLocal(ILocalSymbol symbol)
             {
                 AddKeyword(SyntaxKind.RefKeyword);
                 AddSpace();
+
+                if (symbol.RefKind == RefKind.RefReadOnly)
+                {
+                    AddKeyword(SyntaxKind.ReadOnlyKeyword);
+                    AddSpace();
+                }
```

Comment:
Should the `SymbolDisplay` change be a separate PR, perhaps post 15.5?

Reply:
> Should the SymbolDisplay change be a separate PR, perhaps post 15.5?

Do you have concerns about this change? It feels low risk to me.